src/components/WalletManager.tsx

```diff
@@ -121,7 +125,12 @@ export default function WalletManager() {
           {verifiedAddresses.includes(profile) && <VerifiedIcon />}
         </h1>
         <p className={styles.Address}>
-          {formatAddress(profile)}
+          <Tooltip text="Check address on viewblock">
+            <a href={getViewblockLinkForAddress(profile)} target="_blank">
```

Comment:
This changes the color of the address. The color should be inheriting the color of the parent `p` element.
<img width="363" alt="Screenshot 2022-08-14 at PM 2 42 11" src="https://user-images.githubusercontent.com/30638105/184537370-ef15de9b-3af4-4d51-9aba-8bed257eacb9.png">
